src/docs/sdk/event-payloads/properties/transaction_info.mdx

```diff
@@ -0,0 +1,28 @@
+`transaction_info`
+
+: _Required_. Additional information about the name of the transaction.
+
+```json
+{
+  "transaction_info": {
+    "source": "raw-url"
+  }
+}
+```
+
+`transaction_info.source`
+
+: _Required_. Contains information about how the name of the transaction was determined.
+This will be used by the server to decide whether or not to scrub identifiers
+from the transaction name, or replace the entire name with a placeholder.
+
+| Source             | Description                                                                                                                     | <div style="width:220px">Examples</div>                |
+| ------------------ | ------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------ |
+| `raw-url`          | Raw URL, potentially containing identifiers.                                                                                    | `/auth/login/john123/`<br />`GET /auth/login/john123/` |
+| `raw-url-fallback` | The SDK is configured to send `url-pattern`, but has to fall back to a raw URL for this particular transaction.                 |
+| `url-pattern`      | URL pattern / route                                                                                                             | `/auth/login/:userId/`<br />`GET /auth/login/{user}/`  |
```

Comment:
Could we simplify some of these names, for instance this one to `route` and `raw-url` to just `url`?

Reply:
Sure why not